packages/graphback-runtime-mongodb/src/OffixMongoDBDataProvider.ts

```diff
@@ -10,17 +10,50 @@ import { MongoDBDataProvider } from './MongoDBDataProvider';
  * https://offix.dev/docs/conflict-server#structure-of-the-conflict-error
  */
 export class OffixMongoDBDataProvider<Type = any, GraphbackContext = any> extends MongoDBDataProvider<Type, GraphbackContext> {
+  protected updatedAtField: string;
+  protected createdAtField: string;
 
   public constructor(baseType: GraphQLObjectType, client: any) {
     super(baseType, client);
+    this.createdAtField = undefined;
+    this.updatedAtField = undefined;
+    Object.keys(baseType.getFields()).forEach((k: string) => {
+      if (baseType.getFields()[k]?.astNode?.directives) {
+        baseType.getFields()[k].astNode.directives.forEach((directive: any) => {
+          if (directive?.name?.value === "createdAt") {
+            if (this.createdAtField === undefined) {
+              this.createdAtField = baseType.getFields()[k]?.name;
+            } else {
+              throw Error("Cannot have more than one field with createdAt directive");
```

Comment:
This validation should be handled in SchemaCRUDPlugin. This way it is caught really early before the server starts and the user won't get any surprise when the API is deployed somewhere.